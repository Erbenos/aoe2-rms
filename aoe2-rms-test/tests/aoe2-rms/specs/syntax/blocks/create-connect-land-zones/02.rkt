#lang aoe2-rms
; Connect player 2 and player 4

<LAND-GENERATION>
(create-player-lands
 (terrain-type 'DESERT)
 (land-percent 5))

<CONNECTION-GENERATION>
(create-connect-land-zones
 -6 -8
 (default-terrain-replacement 'ROAD))
