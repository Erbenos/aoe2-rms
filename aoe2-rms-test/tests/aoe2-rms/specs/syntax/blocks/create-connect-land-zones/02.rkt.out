<LAND_GENERATION>
create_player_lands {
  terrain_type DESERT
  land_percent 5
}
<CONNECTION_GENERATION>
create_connect_land_zones -6 -8 {
  default_terrain_replacement ROAD
}
